exercises/proverb/proverb.spec.js

```diff
@@ -0,0 +1,73 @@
+import proverb from './proverb';
+
+
+describe('Proverb xtest Suite', () => {
+  test('a single consequence', () => {
+    const result = proverb('nail', 'shoe');
+
+    expect(result).toEqual(
+      'For want of a nail the shoe was lost.\n' +
+      'And all for the want of a nail.');
+  });
+
+  xtest('a short chain of consequences', () => {
+    const result = proverb('nail', 'shoe', 'horse');
+
+    expect(result).toEqual(
+      'For want of a nail the shoe was lost.\n' +
+      'For want of a shoe the horse was lost.\n' +
```

Comment:
It would make use of a ES6 feature, nice!

Just, be aware of spaces. The multiline string probably needs to be this way:

```javascript
    expect(result).toEqual(
      `For want of a nail the shoe was lost.
For want of a show the horse was lost.
...`
```